core/drivers/atmel_shdwc.c

```diff
@@ -0,0 +1,179 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2015 Atmel Corporation,
+ *                    Nicolas Ferre <nicolas.ferre@atmel.com>
+ * Copyright (c) 2021, Microchip
+ */
+
+#include <at91_clk.h>
```

Comment:
sorry, i missed that.
should be `#include "at91_clk.h"`, below the diamond `#include <>`'s.